common/lib/xmodule/xmodule/js/fixtures/video_all.html

```diff
@@ -29,22 +29,32 @@
         </div>
 
         <div class="focus_grabber last"></div>
-
-        <ul class="wrapper-downloads">
-          <li class="video-tracks">
-            <div class="a11y-menu-container">
-                <a class="a11y-menu-button" href="#" title=".srt">.srt</a>
-                <ol class="a11y-menu-list">
-                  <li class="a11y-menu-item">
-                    <a class="a11y-menu-item-link" href="#txt" title="Text (.txt) file" data-value="txt">Text (.txt) file</a>
-                  </li>
-                  <li class="a11y-menu-item active">
-                    <a class="a11y-menu-item-link" href="#srt" title="SubRip (.srt) file" data-value="srt">SubRip (.srt) file</a>
-                  </li>
-                </ol>
+        
+        <h3 class="hd hd-4 downloads-heading sr" id="video-download-transcripts">Downloads and transcripts</h3>
+        <div class="wrapper-downloads" role="region" aria-labelleby="video-download-transcripts">
+            <div class="wrapper-download-video">
+                <h4 class="hd hd-5">Video</h4>
+                <a class="btn-link video-sources video-download-button" href="#">
+                    Download video file
+                </a>
+            </div>
+            <div class="wrapper-download-transcripts">
+                <h4 class="hd hd-5" id="video-transcripts">Transcripts</h4>
```

Comment:
Please update the fixture file to match the actual file (which is the thing that really sucks about fixture files).